mypy/fastparse.py

```diff
@@ -1008,18 +1013,14 @@ def _visit_implementation(self, node: Optional[ast3.AST]) -> Optional[Type]:
         finally:
             self.node_stack.pop()
 
-    if sys.version_info >= (3, 6):
-        @overload
-        def visit(self, node: ast3.expr) -> Type: ...
+    @overload
+    def visit(self, node: ast3.expr) -> Type: ...
 
-        @overload  # noqa
-        def visit(self, node: Optional[ast3.AST]) -> Optional[Type]: ...
+    @overload  # noqa
+    def visit(self, node: Optional[ast3.AST]) -> Optional[Type]: ...
 
-        def visit(self, node: Optional[ast3.AST]) -> Optional[Type]:  # noqa
-            return self._visit_implementation(node)
-    else:
-        def visit(self, node: Optional[ast3.AST]) -> Any:
-            return self._visit_implementation(node)
+    def visit(self, node: Optional[ast3.AST]) -> Optional[Type]:  # noqa
+        return self._visit_implementation(node)
```

Comment:
Also I think `_visit_implementation` only existed because of conditional definition, you can kill it now.

Reply:
:heavy_check_mark: